florestad/src/json_rpc/server.rs

```diff
@@ -197,15 +197,31 @@ impl Rpc for RpcImpl {
             }
         };
         let node = ip.parse().map_err(|_| Error::InvalidAddress)?;
-        self.node.connect(node, port).unwrap();
-        Ok(true)
+
+        match self.node.connect(node, port) {
+            Ok(_) => return Ok(true),
```

Comment:
Clippy is complaining because of this `return` state. You can return this value by just leaving it as the last statement, like this:
```rust
            Ok(_) => Ok(true),
```